.gitignore

```diff
@@ -1,30 +0,0 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Compiled builds
-block/build
```

Comment:
@buchansm this repo now auto builds a branch with the same name ending in `--built` in this case `buchansm/PRS-794-reducing-unused-javascript--built` 

For that reason you should no longer need to remove the gitignore nor commit the build directory.

Reply:
Also I noticed there are unrelated commits that made it in this repo. Can you rebase and only bring in the relevant commits?